hs_core/templates/debug/debug_resource.html

```diff
@@ -129,15 +129,15 @@ <h2>iRODs Issues</h2>
       else {
         // Failed to retrieve data
         const title = document.createElement("P");
-        const titleText = document.createTextNode("Failed to get iRODS issues.");
+        const titleText = document.createTextNode("Failed to get S3 issues.");
         title.appendChild(titleText);
-        irodsIssuesDiv.appendChild(title);
+        s3IssuesDiv.appendChild(title);
         // Remove loading indicator
-        document.querySelector("#irods-issues > .loading").remove();
+        document.querySelector("#s3-issues > .loading").remove();
       }
     }
   };
-  xhttp.open("GET", "irods-issues/", true);
+  xhttp.open("GET", "s3-issues/", true);
```

Comment:
does this `debug_resource` page still load? Just curious...I guess we can test it on beta...

Reply:
It should, I remember when I made these changes I updated the url on the backend first and then made sure it was updated in the frontend.